apps/console/src/features/server/components/remote-logging-config-form.tsx

```diff
@@ -35,27 +38,31 @@ import React, { ReactElement, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useDispatch } from "react-redux";
 import { Dispatch } from "redux";
-import { 
+import {
     restoreRemoteLogPublishingConfigurationByLogType,
     updateRemoteLogPublishingConfigurationByLogType
 } from "../api/server";
 import { LogType, RemoteLogPublishingConfigurationInterface } from "../models/server";
 
 interface RemoteLoggingConfigFormProps extends IdentifiableComponentInterface {
+    formId: "audit" | "carbon";
```

Comment:
Lets remove this.

Reply:
fixed with https://github.com/wso2/identity-apps/pull/5183/commits/bb554588d1bad946890ec7690bdf5e791370ee7e